### DISCUSSION TITLE
[2024-01-31] This Bi-Weekly in Dora-rs

### DISCUSSION THREAD

**Discussion by haixuanTao** [dora-rs/dora#422] [Status Update]
# Quick Demo dora-rs on Mineflyer

- This demo will show the current progress for dora-rs on mineflyer

**Comment by haixuanTao** [comment]
# Released a new dora version `0.3.2` 
  - ROS2 Array fixes
  - Simpler logging environment that do not need to use the UUID.
  - Single binary
  - Simplified installation
  - Updated documentation: https://dora.carsmos.ai/

Thanks https://github.com/GoAfteraDeer for pointing out and testing out the error within ROS2 Array.

**Reply by put-down-crate** [reply]
I translate done new text for 0.3.2, in crowdin.com.
I have some think In README, may be we can, if have:
~~1.confirm or define some basic concept in dora, e.g. what is node what is operator, this will help sure more real intent  in translate progress.~~
2.help more people know or growing dora if add detail or trend for roadmap or plan to README.

**Reply by haixuanTao** [reply]
Thanks!

**Reply by haixuanTao** [reply]
Sure thing! We probably have to update the README

**Reply by put-down-crate** [reply]
i done again, text "0.3.2" to "0.3.2" 
:-)

**Comment by haixuanTao** [comment]
# Community

- Preparing a talk for FOSDEM with a live demo of hot-reloading on RoboMaster and Mistral LLM.
- Investigating a Rust Conference in Paris for the scientific community ( AI, ... ). End of next Sept-Dec. Beginning of next year. Currently looking for volunteers.
- Talked with Lee-Orr who is working on a generic solution for hot reloading applications in Rust. See: https://github.com/users/lee-orr/projects/1/views/6 and https://excalidraw.com/#json=CkmlX86XKPbwtx0zbCgMi,MK33jIwkSJj7i-taIa8i6w

**Reply by linyihai** [reply]
Thanks for your share.
